util/debug.go

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) OpenFaaS Project 2017. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+package util
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/morikuni/aec"
+)
+
+var useDockerCLIWarningDisplayed bool
+
+func IsDebugEnabled() bool {
+	val, exists := os.LookupEnv("debug")
+	return exists && (val == "1" || val == "true")
```

Comment:
Why 1 or true?

Reply:
Let's not add this `debug`helper method - we will be looking at logging across the board, so let's save it for later. Please try to only change one thing or add one thing at a time. It just ends up making the PRs drag out unnecessarily.

Reply:
This comes from builder package, I would like to use just the same thing in commands, I can duplicate the debug() in commands package in the meantime, instead of creating a util package to have common code.